File: PRACTICA/app/src/main/java/com/example/neurocare/PacientiFragment.java
package com.example.neurocare;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;
import java.util.Arrays;
import java.util.List;

public class PacientiFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_pacienti, container, false);

        RecyclerView recyclerView = view.findViewById(R.id.recyclerView);
        List<String> pacienti = Arrays.asList(
                "Popescu Ionut", "Iota Cristina", "Adam Cristina", "Munteanu Andreea",
                "Mitroi Adina", "Vlad Dumitru", "Enache Cristian"
        );

        recyclerView.setLayoutManager(new LinearLayoutManager(requireContext()));
        recyclerView.setAdapter(new PacientAdapter(pacienti));

        return view;
    }
}

File: myproject-main/app/src/main/java/com/example/mhealthapplication/PacientiFragment.kt
package com.example.mhealthapplication

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class PacientiFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        var view : View = inflater.inflate(R.layout.fragment_pacienti, container, false)
        val recyclerView = view.findViewById<RecyclerView>(R.id.recyclerView)

        // Sample data
        val examList: MutableList<Item> = ArrayList()
        examList.add(Item("Popescu Ionut"))
        examList.add(Item("Iota Cristina"))
        examList.add(Item("Adam Cristina"))
        examList.add(Item("Munteanu Andreea"))
        examList.add(Item("Mitroi Adina"))
        examList.add(Item("Vlad Dumitru"))
        examList.add(Item("Enache Cristian"))

        // Set LayoutManager
        recyclerView.layoutManager = LinearLayoutManager(requireContext())

        // Set Adapter
        val adapter = MyAdapter(examList)
        recyclerView.adapter = adapter
        return view
    }
}